execution.go

```diff
@@ -19,8 +19,8 @@ import (
 	"github.com/vektah/gqlparser/v2/gqlerror"
 )
 
-func newExecutableSchema(plugins []Plugin, maxRequestsPerQuery int64, services ...*Service) *ExecutableSchema {
-	client := NewClient()
+func newExecutableSchema(plugins []Plugin, maxRequestsPerQuery, maxResponseSize int64, services ...*Service) *ExecutableSchema {
```

Comment:
might be worth reconsidering the signature of this function? perhaps give it a client that is already configured rather than expanding the argument list to include options for a client.

Reply:
agreed, you should try avoid passing arguments to a function just so it can pass those along. Like Adam said, passing the already configured client in would be more clear.